api/nakadi-event-bus-api.yaml

```diff
@@ -743,6 +743,62 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /subscriptions:
+    post:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        This endpoint creates a Subscription for EventType(s). The subscription is needed to be able to consume
```

Comment:
I think it would ok to have it in plural without parenthesis. In a programatic point of view it makes sense to refer to a collection of 1 as a collection.